src/did/Did.utils.ts

```diff
@@ -27,7 +27,10 @@ export function decodeDid(
   identifier: string,
   encoded: QueryResult
 ): IDid | null {
-  const json = encoded && encoded.encodedLength ? encoded.toJSON() : null
+  const json =
+    encoded && encoded.encodedLength && !encoded.isEmpty
```

Comment:
```suggestion
    encoded && !encoded.isEmpty
```
does `encoded.encodedLength` serve any other function besides what `!encoded.isEmpty` does? It seems to me that the former was also meant to assure empty QueryResults are not decoded and null is returned instead, which only the latter reliably does. In that case, we could remove the former?
@tjwelde